gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -345,3 +353,34 @@ nnsconf_get_fullpath (const gchar * subpluginname, nnsconf_type type)
   g_free (filename);
   return ret;
 }
+
+/** @brief Public function defined in the header */
+const gchar *
+nnsconf_get_value (nnsconf_type_value type)
+{
+  int i;
+  gchar *ret;
+
+  switch (type) {
+    case NNSCONF_VAL_TF_MEM_OPTMZ:
+      for (i = 0; i < CONF_SOURCES; i++) {
+        if (conf.valueTF_MEM_OPTMZ[i]) {
+          conf.retTF_MEM_OPTMZ = g_malloc_n (1, sizeof (gchar *));
```

Comment:
``` sizeof (gchar *)``` ==> ```sizeof(gchar)```

Anyway, as this is static, this is better (Line 52):
```
gchar retTF_MEM_OPTMZ[1];
```
As the size is 1, this is even better:
```
gchar retTF_MEM_OPTMZ;
```

Even much better, having two funcs:
```const gchar *nnsconf_get_value_str()``` and
```gboolean nnsconf_get_value_bool()``` would be much better and extensible.
with ```gboolean retTF_MEM_OPTMZ;```

Reply:
If ```const gchar *nnsconf_get_value_str()``` is called for a bool variable, you may return with static const variable of ```static const gchar *true = "true", *false = "false"; ```

Reply:
@myungjoo 
First looks better for me.
Don't we consider other types? like `int`, `float` and the others which we deal with.

Reply:
You can make them whenever they are needed :) (We just don't have them, yet)